### PR TITLE
Tweaks on chat

### PR DESCRIPTION
**Pull request by cecileboucheron**

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
The commits 560c576efd04...644063d86f6d introduce visual changes on linux. <details><summary>:mag_right: 6 changed</summary>

 * Changed: **Text_Message-Normal_Text** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Text_Message-Normal_Text-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Text_Message-Normal_Text-644063d86f6d.png)  
   Text_Message-Normal_Text comparison
 * Changed: **Stacked_Text_Message-Stacked_two_messages_one_edited** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_two_messages_one_edited-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_two_messages_one_edited-644063d86f6d.png)  
   Stacked_Text_Message-Stacked_two_messages_one_edited comparison
 * Changed: **Stacked_Text_Message-Stacked_two_messages** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_two_messages-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_two_messages-644063d86f6d.png)  
   Stacked_Text_Message-Stacked_two_messages comparison
 * Changed: **Stacked_Text_Message-Stacked_someone_else_two_sent** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_someone_else_two_sent-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_someone_else_two_sent-644063d86f6d.png)  
   Stacked_Text_Message-Stacked_someone_else_two_sent comparison
 * Changed: **Stacked_Text_Message-Stacked_one_sent_one_pending** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_one_sent_one_pending-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_one_sent_one_pending-644063d86f6d.png)  
   Stacked_Text_Message-Stacked_one_sent_one_pending comparison
 * Changed: **Stacked_Text_Message-Stacked_one_sent_one_failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_one_sent_one_failed-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495676689655-560c576efd04-644063d86f6d-linux/Stacked_Text_Message-Stacked_one_sent_one_failed-644063d86f6d.png)  
   Stacked_Text_Message-Stacked_one_sent_one_failed comparison
</summary>

File: shared/chat/conversation/messages/shared.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import {globalColors} from '../../../styles'
 
 const marginColor = (user: string, isYou: boolean, isFollowing: boolean, isBroken: boolean) => {
   if (isYou) {
-    return globalColors.white
+    return 'transparent'
```
Comment (chromakode) [inline shared/chat/conversation/messages/shared.js]: `globalColors.transparent`?

**Comment by keybase-ci-visdiff** [comment]
The commits 560c576efd04...0a5eb66bb85a introduce visual changes on linux. <details><summary>:mag_right: 6 changed</summary>

 * Changed: **Text_Message-Normal_Text** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Text_Message-Normal_Text-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Text_Message-Normal_Text-0a5eb66bb85a.png)  
   Text_Message-Normal_Text comparison
 * Changed: **Stacked_Text_Message-Stacked_two_messages_one_edited** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_two_messages_one_edited-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_two_messages_one_edited-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_two_messages_one_edited comparison
 * Changed: **Stacked_Text_Message-Stacked_two_messages** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_two_messages-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_two_messages-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_two_messages comparison
 * Changed: **Stacked_Text_Message-Stacked_someone_else_two_sent** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_someone_else_two_sent-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_someone_else_two_sent-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_someone_else_two_sent comparison
 * Changed: **Stacked_Text_Message-Stacked_one_sent_one_pending** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_one_sent_one_pending-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_one_sent_one_pending-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_one_sent_one_pending comparison
 * Changed: **Stacked_Text_Message-Stacked_one_sent_one_failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_one_sent_one_failed-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495751719827-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_one_sent_one_failed-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_one_sent_one_failed comparison
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 560c576efd04...0a5eb66bb85a introduce visual changes on linux. <details><summary>:mag_right: 6 changed</summary>

 * Changed: **Text_Message-Normal_Text** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Text_Message-Normal_Text-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Text_Message-Normal_Text-0a5eb66bb85a.png)  
   Text_Message-Normal_Text comparison
 * Changed: **Stacked_Text_Message-Stacked_two_messages_one_edited** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_two_messages_one_edited-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_two_messages_one_edited-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_two_messages_one_edited comparison
 * Changed: **Stacked_Text_Message-Stacked_two_messages** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_two_messages-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_two_messages-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_two_messages comparison
 * Changed: **Stacked_Text_Message-Stacked_someone_else_two_sent** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_someone_else_two_sent-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_someone_else_two_sent-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_someone_else_two_sent comparison
 * Changed: **Stacked_Text_Message-Stacked_one_sent_one_pending** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_one_sent_one_pending-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_one_sent_one_pending-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_one_sent_one_pending comparison
 * Changed: **Stacked_Text_Message-Stacked_one_sent_one_failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_one_sent_one_failed-560c576efd04.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1495759063179-560c576efd04-0a5eb66bb85a-linux/Stacked_Text_Message-Stacked_one_sent_one_failed-0a5eb66bb85a.png)  
   Stacked_Text_Message-Stacked_one_sent_one_failed comparison
</summary>

**Pull request merged**